search.py

```diff
@@ -27,24 +22,46 @@ def main(user_query, browser):
     soup_link = BeautifulSoup(requests_result.content, "html.parser")
     links = soup_link.find_all("a")
 
-    result = open("results.txt", "w")
+    my_file = Path(output)
+
+    if my_file.exists():
+        to_file(links, my_file)
+    else:
+        to_console(links)
+
+
+def to_file(links, my_file):
+    result = open(my_file, "w")
+    result.write("Title,URL\n")
+
+    add_results(links, result)
+
+    print("Done!")
+
 
-    print("Pushing results in file -> results.txt...")
+def to_console(links):
+    result = open('result.csv', "w")
```

Comment:
do not use useless writing and reading results from/to the file. 
find another approach